sphinx/writers/latex.py

```diff
@@ -1377,10 +1381,15 @@ def visit_row(self, node):
                 break
             else:  # a bottom of multirow cell
                 self.table.col += cell.width
-                if cell.col != 0:
+                if cell.col:
                     self.body.append('&')
-                if cell.width > 1:  # use \multicolumn for wide multirow cell
-                    self.body.append('\\multicolumn{%d}{|l|}{}\\relax ' % cell.width)
+                if cell.width == 1:
+                    # insert suitable strut for equalizing row heights in given multirow
+                    self.body.append('\\sphinxtablestrut{%d}' % cell.cell_id)
+                else:  # use \multicolumn for wide multirow cell
+                    self.body.append('\\multicolumn{%d}{|l|}'
+                                     '{\\sphinxtablestrut{%d}}\\relax ' %
```

Comment:
this ``\\relax`` is from earlier code and presumably with the idea to "absorb" a possible space or end-of-line space which might follow. But it looks superfluous because the expansion of LaTeX 's ``\\multicolumn`` will end with insertion of an ``\\ignorespaces`` anyhow. Hence I think it can be removed.